packages/react-instantsearch-core/src/components/Configure.tsx

```diff
@@ -5,7 +5,17 @@ import type { UseConfigureProps } from '../connectors/useConfigure';
 export type ConfigureProps = UseConfigureProps;
 
 export function Configure(props: ConfigureProps) {
-  useConfigure({ ...props }, { $$widgetType: 'ais.configure' });
+  useConfigure(
+    Object.keys(props).reduce<ConfigureProps>((acc, key) => {
```

Comment:
Worth leaving a comment here about Sentry being the cause of this issue for future readers?